src/lib/index.js

```diff
@@ -350,19 +350,25 @@ lib.noneOrAll = function(containerIn, containerOut, attrList) {
     }
 };
 
-/** merge data array into calcdata items
+lib.mergeArray = function(traceAttr, cd, cdAttr) {
```

Comment:
🎎 with a description of the difference between `mergeArray` and `fillArray`? And perhaps some indication of *why* one would use merge vs fill? Makes me wonder whether any of our other existing merge usages should be fill...

`mergeArray` doesn't need `fn`?